src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString.split(';');
+  const trimed = properties.map(prop =>
+    prop.split(':').map(elem => elem.trim()));
```

Comment:
You can do it without extra variable
```suggestion
  const properties = sourceString
  	.split(';')
  	.map(prop =>
    	prop.split(':').map(elem => elem.trim())
    );
```